Packs/Malwarebytes/Integrations/Malwarebytes/Malwarebytes.py

```diff
@@ -0,0 +1,1516 @@
+import demistomock as demisto
+from CommonServerPython import *
+from CommonServerUserPython import *
+""" IMPORTS """
+
+import requests
+from requests_oauthlib import OAuth2Session
+from oauthlib.oauth2 import BackendApplicationClient
+import json
+import sys
+import traceback
+import time
+from typing import Dict
+
+# Disable insecure warnings
+requests.packages.urllib3.disable_warnings()
+
+
+# Get OAuth2 token
+def NEBULA_URL(path):
+    return "{NEBULA_URL}{PATH}".format(NEBULA_URL="https://cloud.malwarebytes.com", PATH=path)
+
+
+def get_nebula_client(client_id, client_secret, account_id):
+    client_scope = ["read", "write", "execute"]
+    headers = {"x-mwb-clientid": client_id,
+               "x-mwb-accountid": account_id}
+
+    client = BackendApplicationClient(client_id, scope=client_scope)
+    nebula = OAuth2Session(client=client, scope=client_scope)
+    nebula.headers.update(headers)
+    token = nebula.fetch_token(
+        token_url=NEBULA_URL('/oauth2/token'),
+        client_secret=client_secret, scope=client_scope)
+
+    return "Bearer " + token['access_token']
+
+
+# Test connectivity to the Nebula cloud
+def test_connectivity(account_id, client_id, auth_token):
+
+    url = 'https://cloud.malwarebytes.com/api/v2/endpoints'
+
+    headers = {
+        'X-MWB-AccountID': account_id,
+        'X-MWB-ClientID': client_id,
+        'Content-Type': "application/json",
+        'Authorization': auth_token
+    }
+    response = requests.request("GET", url, headers=headers)
+
+    if response.status_code == 200:
+        return True
+    else:
+        return_error(message="HTTP Error: " + str(response.status_code) + " on testing connectivity", error='')
+
+
+# Get all the endpoints from the Malwarebytes Cloud
+def get_all_endpoints(account_id, client_id, auth_token):
+
+    url = 'https://cloud.malwarebytes.com/api/v2/endpoints'
+
+    headers = {
+        'X-MWB-AccountID': account_id,
+        'X-MWB-ClientID': client_id,
+        'Content-Type': "application/json",
+        'Authorization': auth_token
+    }
+    response = requests.request("GET", url, headers=headers)
+
+    if response.status_code == 200:
+
+        machine = response.text
+        machine = json.loads(machine)
+        machines = machine['machines']
+        cursor = machine['next_cursor']
+        while cursor != "":
+            cursor_temp, machines_temp = get_all_endpoints_paginated(account_id, client_id, auth_token, cursor)
+            machines.extend(machines_temp)
+            cursor = cursor_temp
+        return machines
+    else:
+        return_error(message="HTTP Error: " + str(response.status_code) + " on getting All Endpoints", error='')
```

Comment:
avoid using `return_error` in the code, as it should be called only once in the `try-except` in the `main` function
raise exceptions instead